src/org/labkey/targetedms/model/RawMetricDataSet.java

```diff
@@ -356,6 +356,34 @@ public boolean isValueOutlier(GuideSetStats stat)
         }
     }
 
+    public boolean isMeanDeviationOutlier(GuideSetStats stat)
```

Comment:
See previous comment. I think this can be folded in as a third case in `isValueOutlier()`